bitnami/redmine/values.yaml

```diff
@@ -600,280 +717,240 @@ mailReceiver:
+  ## @param mailReceiver.podAffinityPreset Pod affinity preset. Ignored if `affinity` is set. Allowed values: `soft` or `hard`
   ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity
   ## Allowed values: soft, hard
   ##
   podAffinityPreset: ""
-
-  ## Pod anti-affinity preset
+  ## @param mailReceiver.podAntiAffinityPreset Pod anti-affinity preset. Ignored if `affinity` is set. Allowed values: `soft` or `hard`
   ## Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity
   ## Allowed values: soft, hard
   ##
   podAntiAffinityPreset: soft
-
-  ## Node affinity preset
+  ## @param mailReceiver.nodeAffinityPreset.type Node affinity preset. Ignored if `affinity` is set. Allowed values: `soft` or `hard`
+  ## @param mailReceiver.nodeAffinityPreset.key Node label key to match. Ignored if `affinity` is set.
+  ## @param mailReceiver.nodeAffinityPreset.values Node label values to match. Ignored if `affinity` is set.
   ## Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity
-  ## Allowed values: soft, hard
   ##
   nodeAffinityPreset:
-    ## Node affinity type
     ## Allowed values: soft, hard
     ##
     type: ""
     ## Node label key to match
     ## E.g.
-    ## key: "kubernetes.io/e2e-az-name"
     ##
+    # key: "kubernetes.io/e2e-az-name"
     key: ""
     ## Node label values to match
     ## E.g.
-    ## values:
-    ##   - e2e-az1
-    ##   - e2e-az2
     ##
+    # values:
+    #   - e2e-az1
+    #   - e2e-az2
     values: []
-
-  ## Affinity for pod assignment
+  ## @param mailReceiver.affinity Affinity for pod assignment
   ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
   ## Note: podAffinityPreset, podAntiAffinityPreset, and  nodeAffinityPreset will be ignored when it's set
   ##
   affinity: {}
-
-  ## Node labels for pod assignment. Evaluated as a template.
+  ## @param mailReceiver.nodeSelector Node labels for pod assignment
   ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
   ##
   nodeSelector: {}
-
-  ## Tolerations for pod assignment
+  ## @param mailReceiver.tolerations Tolerations for pod assignment
   ## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
   ##
   tolerations: []
 
-## volumePermissions: Change the owner and group of the persistent volume mountpoint to runAsUser:fsGroup values from the securityContext section.
-##
-volumePermissions:
-  enabled: false
-  image:
-    registry: docker.io
-    repository: bitnami/bitnami-shell
-    tag: 10-debian-10-r112
-    pullPolicy: Always
-    ## Optionally specify an array of imagePullSecrets.
-    ## Secrets must be manually created in the namespace.
-    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
-    ##
-    pullSecrets: []
-    ##   - myRegistryKeySecretName
```

Comment:
Why are you deprecating this image?

Reply:
Because now we are changing the ownership to `redmine:root`, and the `bitnami-shell` image does not have the `redmine` user defined and we do not want to hardcode its UID.

Reply:
Fair enough

Reply:
Please remember to update the pipeline to remove `bitnami-shell` as dependency.